app/models/profile.rb

```diff
@@ -93,6 +93,7 @@ def activate(reason_deactivated: nil)
 
     now = Time.zone.now
     is_reproof = Profile.find_by(user_id: user_id, active: true)
+    is_biometric = biometric?(idv_level) && !biometric?(is_reproof.idv_level) if is_reproof
```

Comment:
seems weird to me to mix `&&` and trailing `if` on the same line, what about:
```suggestion
    is_biometric = biometric?(idv_level) && (!is_reproof || biometric?(is_reproof.idv_level))
```

Reply:
also cc @18F/identity-ada to get some eyes on this section to make sure this is the right spot to instrument to record a "legacy IDV upgrade"

Reply:
I don't this this logic would work. If its a new biometric comparison `!is_reproof` would be true. I think we could do `is_biometric = biometric?(idv_level) && (is_reproof ^ biometric?(is_reproof.idv_level))` I believe with "^" as control flow only one can be true. If both are true, it will be false.

Reply:
I should probably call it `is_biometric_upgrade` to prevent confusion. I'm not tracking all biometric upgrades.